addons/web/static/src/views/calendar/calendar_common/calendar_common_renderer.js

```diff
@@ -199,6 +199,9 @@ export class CalendarCommonRenderer extends Component {
         const record = this.props.model.records[event.id];
 
         if (record) {
+            if (this.env.searchModel?.context?.default_calendar_event_id == event.id) {
```

Comment:
```suggestion
            if (this.env.searchModel?.context?.default_calendar_event_id === event.id) {
```

Reply:
Hello @leonardopavanrocha ,
I made the changes please check this out!
Thanks!